core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -126,11 +135,12 @@
   }
 
   /**
-   * Returns the SslSocketFactory singleton, which can be used to create SslSockets to Cloud SQL.
+   * Returns the CoreSocketFactory singleton, which can be used to createSslSocket SslSockets to
+   * Cloud SQL.
    *
-   * @return the SslSocketFactory singleton.
+   * @return the CoreSocketFactory singleton.
```

Comment:
You don't need this if it simply repeats what's in the summary.

Reply:
Removed.